packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -123,7 +123,6 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
         <FocusZone>
           <table
```

Comment:
Not sure if I understand the concern correctly - these roles are not dependent as far as I know. I am using application to switch screen reader mode (which we need). I am no longer using `role=presentation` as you suggested and it makes sense to keep the table roles intact.